src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -144,22 +160,52 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 			return;
 		}
 
-		if (TestDataGenerator.getInstance().isTestConversation(conversation.getId())) {
-			// No user may contribute to a test conversation
-			response.sendRedirect("/chat/" + conversationTitle);
-			return;
+		Part messagePart = request.getPart("message");
+		if (messagePart != null) {
+			Scanner s = new Scanner(messagePart.getInputStream());
+
+			if (s.hasNextLine()) {
+				String messageContent = s.nextLine();
+
+				// allows users to enter basic HTML tags that are not a threat to security
+				String HTMLMessageContent = clean(messageContent);
```

Comment:
Nice touch with adding this!

Reply:
Are you referring to the HTML tags? I want to shout out to @LesC1210 , who implemented that in #8 :tada: